docs/README.md

```diff
@@ -93,6 +93,8 @@ The following options can be enabled/disabled via the search configuration.
 - `"facets": true|false|['match-type' => "all" (default)|...]`
 - `"woocommerce": true|false (default)`
 - `"autosuggest": true|false (default)`
+- `"users": (default) true|false`
+- `"terms": (default) true|false`
```

Comment:
Is the `(default)` in the position that it is to indicate that `true` is the default? It looks a little weird when all the other instances have `(default)` placed at the end of the line. Maybe `true (default)|false`?